### DISCUSSION TITLE
F1 API 2024 Schedule

### DISCUSSION THREAD

**Discussion by HairyOtter07** [theOehrly/Fast-F1#611] [Q&A] [answered]
Hi, I was working on a personal project using the F1 API and I encountered an annoying hiccup that I was hoping you could help me with. When I go to the season index at https://livetiming.formula1.com/static/2024/Index.json it only responds with Spain, Austria, and now Silverstone. Earlier in the year, it was showing all of the races up to that point in the season, and if I go to the endpoints for those races I can still access their data, which is why I'm confused by the change. Could this be related to the server change you mentioned with regards to the 2022 issues? And how does FastF1 handle it, does it just have a cache or something? Any help or advice would be appreciated, thanks in advance :)

**Comment by theOehrly** [comment, marked as answer]
Yes, this is most certainly caused by the server changes because the timing coincides with that. Some things have improved again already, so this may well be fixed too.
FastF1 by default uses its own schedule backend which is built by aggregating data from other sources.
You will only notice this in FastF1 if you were to force usage of the livetiming API for schedule information (by passing backend='f1timing' for the relevant schedule related function calls).

**Comment by HairyOtter07** [comment]
That makes a lot of sense, hoping F1 fixes the livetiming issues soon. Thank you!
